packages/jellyfish-wallet-mnemonic/src/hd_node.ts

```diff
@@ -53,6 +53,14 @@ export class MnemonicHdNode implements WalletHdNode {
     return node.publicKey
   }
 
+  /**
+   * @return Promise<Buffer> uncompressed public key
+   */
+  async unPublicKey (): Promise<Buffer> {
+    // set an empty Buffer due to unimplemented on bip32.BIP32Interface
+    return Buffer.alloc(0)
```

Comment:
You should throw an error instead, to prevent sending to wrong address. Since `0x0` on EVM is a valid address.